packages/block-library/src/more/edit.js

```diff
@@ -40,17 +44,34 @@ export default function MoreEdit( {
 	return (
 		<>
 			<InspectorControls>
-				<PanelBody>
-					<ToggleControl
-						__nextHasNoMarginBottom
-						label={ __(
-							'Hide the excerpt on the full content page'
-						) }
-						checked={ !! noTeaser }
-						onChange={ toggleHideExcerpt }
-						help={ getHideExcerptHelp }
-					/>
-				</PanelBody>
+				<ToolsPanel
+					label={ __( 'Settings' ) }
+					resetAll={ () => {
+						setAttributes( {
+							noTeaser: false,
+							customText: DEFAULT_TEXT,
```

Comment:
I think this should reset `customText`; the settings have no connection to it.

Reply:
Hey @Mamaduka,

I have fixed this. Please check it now.